test/drivers/test_suites/syscall_exit_suite/mknod_e.cpp

```diff
@@ -0,0 +1,43 @@
+#include "../../event_class/event_class.h"
+#if defined(__NR_mknod)
+#include <sys/sysmacros.h>
+TEST(SyscallEnter, mknodE)
+{
+	auto evt_test = get_syscall_event_test(__NR_mknod, ENTER_EVENT);
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL  ===========================*/
+
+	char path[] = "/tmp/";
+
+	uint32_t mode = 0060000 | 0666;
+	uint32_t dev = 61440;
+	assert_syscall_state(SYSCALL_SUCCESS, "mknod", syscall(__NR_mknod, (void *)(path), (mode_t)mode, (dev_t)dev));
```

Comment:
the same call fails on other tests :thinking: 
```suggestion
	assert_syscall_state(SYSCALL_FAILURE, "mknod", syscall(__NR_mknod, (void *)(path), (mode_t)mode, (dev_t)dev));
```